src/ConnectionCard/ConnectionCard.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Badge } from '../Badge';
+import { Price } from '../Price';
+import ConnectionCardRow from './ConnectionCardRow';
+import TripSector from './TripSector';
+import BadgesContainer from './BadgesContainer';
+import type { BadgeProps } from '../Badge/BadgeTypes';
+import type { PriceProps } from '../Price/PriceTypes';
+import type { TripSectorProps } from './ConnectionCardTypes';
+import StyleSheet from '../PlatformStyleSheet';
+
+type TripSectorWithId = { ...TripSectorProps, id: number };
+type BadgeWithId = { ...BadgeProps, id: number };
+
+type Props = {|
+  padding?: boolean,
+  wayForth: Array<TripSectorWithId>,
+  wayBack?: Array<TripSectorWithId>,
+  price: PriceProps,
+  duration?: string,
+  badges?: Array<BadgeWithId>,
+|};
```

Comment:
read-only https://flow.org/en/docs/types/interfaces/#toc-interface-property-variance-read-only-and-write-only